sdk/go/common/testing/diagtest/sink.go

```diff
@@ -39,3 +40,14 @@ func LogSink(t testing.TB) diag.Sink {
 		},
 	)
 }
+
+func TestSink(stdout, stderr *bytes.Buffer) diag.Sink {
```

Comment:
I swear someone added MockSink in a PR like the other day?

Reply:
https://github.com/pulumi/pulumi/pull/17312/files#diff-f204e4f3d30e81d214339b07ea98c7a5ccc90167b3db46253b720bf70b0fdee0R44 it's not merged yet.  I guess it'll conflict depending on who merges first.  I can rename it to `MockSink` though.

Reply:
👍